.dotnet/src/Custom/VectorStores/VectorStoreCreationOptions.cs

```diff
@@ -1,17 +1,17 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 
 namespace OpenAI.VectorStores;
 
 [CodeGenModel("CreateVectorStoreRequest")]
 public partial class VectorStoreCreationOptions
 {
     /// <summary> A list of [File](/docs/api-reference/files) IDs that the vector store should use. Useful for tools like `file_search` that can access files. </summary>
-    public IList<string> FileIds { get; init; }
+    public IList<string> FileIds { get; set; }
```

Comment:
This one is a collection.

Reply:
Let's fix it in a separate PR - https://github.com/joseharriaga/openai-in-typespec/pull/182#discussion_r1693640061